Source/Core/Core/IOS/Network/IP/Top.cpp

```diff
@@ -927,8 +925,9 @@ IPCCommandResult NetIPTop::HandleRecvFromRequest(const IOCtlVRequest& request)
 IPCCommandResult NetIPTop::HandleGetAddressInfoRequest(const IOCtlVRequest& request)
 {
   addrinfo hints;
+  bool hints_valid = request.in_vectors.size() > 2 && request.in_vectors[2].size;
```

Comment:
Could make this `const` while you're at it.
```suggestion
  const bool hints_valid = request.in_vectors.size() > 2 && request.in_vectors[2].size;
```